src/sentry/web/frontend/debug/mail.py

```diff
@@ -362,7 +366,7 @@ def digest(request):
         state["groups"][group.id] = group
 
         offset = timedelta(seconds=0)
-        for i in range(random.randint(1, 10)):
+        for j in range(random.randint(1, 10)):
```

Comment:
If the `i` and `j` are bothering you, you can do:
```py
for _ in range(random.randint(1, 30)):
	...
    for _ in range(random.randint(1, 10)):
```

Reply:
Good call, just added.